app/src/main/java/ch/sheremet/dasfoo/rover/android/client/grpc/authentication/GrpcProviderInstaller.java

```diff
@@ -0,0 +1,28 @@
+package ch.sheremet.dasfoo.rover.android.client.grpc.authentication;
+
+import android.content.Intent;
+
+import com.google.android.gms.security.ProviderInstaller;
+
+/**
+ * Created by Katarina Sheremet on 5/31/16 3:22 PM.
+ */
+public class GrpcProviderInstaller implements ProviderInstaller.ProviderInstallListener {
```

Comment:
Naming: this is just a listener for `ProviderInstaller`; name has to be reflect the intent.